### DISCUSSION TITLE
Private command appear in completion suggestion

### DISCUSSION THREAD

**Discussion by vdesabou** [bashly-framework/bashly#387] [General]
Hi,

I have several private commands and they appear in the list of completion suggestions, I was wondering if this was expected or not?

**Comment by DannyBen** [comment]
Not expected. Private commands / flags should not be generated in completions.
Do you wish to create an issue for it, or should I?

**Reply by DannyBen** [reply]
Converted to issue #388

**Comment by DannyBen** [comment]
This is fixed now in the master branch and the edge docker.
@vdesabou - if you want to test it, you can [follow one of these options](https://github.com/DannyBen/bashly/tree/master/edge#readme).

I will release a version after you test or towards the end of the week - whichever comes first.

Thanks for reporting.

**Reply by vdesabou** [reply]
Your fix works perfectly, thanks !
